### ISSUE TITLE
place of Runs

### ISSUE THREAD

**Issue by LipeDuoli** [speedruncom/api#65]
I trying to get something like latest runs in speedrun.com homepage.
Using this query
https://www.speedrun.com/api/v1/runs?status=verified&orderby=submitted&direction=desc&embed=game
I get the list of the  Latest Runs but i don't have the place of each run.

there some way to do this and get the run place too?

Thanks

**Comment by geovannyAndrew** [comment]
I have the same trouble and I haven't found a solution for this.

**Comment by Duxez** [comment]
I mean, you get the weblink to the run, you get the game id.
What do you guys mean with "run place"

**Comment by LipeDuoli** [comment]
run place is the leaderboard position of each run returned...

**Comment by Duxez** [comment]
In the game data you get with the url you posted there's an object named links, it holds the route to the leaderboard of the run.

**Comment by LipeDuoli** [comment]
Yes, I can make another query and get everything.. but will be nice if I get the position already when I make the query for latestruns... or maybe an option to embed the current leaderboard position of that Run, not the entire leaderboard...

**Comment by geovannyAndrew** [comment]
Yes, the problem is if you want to show a list of lastest runs with this "place" you'd have to make a request for each run, this could be problematic. Thanks.
